README.md

```diff
@@ -27,8 +27,9 @@ If you're a web dapp developer, we've got two types of guides for you:
 ## Building locally
 
  - Install [Node.js](https://nodejs.org/en/) version 8.11.3 and npm version 6.1.0
- - Install dependencies:
-   - If you are using nvm (recommended) running `nvm use` will automatically choose the right node version for you.
+   - If you are using [nvm](https://github.com/creationix/nvm#installation) (recommended) running `nvm use` will automatically choose the right node version for you.
+   - Update npm to 6.1.0: ```npm install -g npm@6.1.0```
```

Comment:
Not sure if this is necessary since it's already mentioned [here](https://github.com/MetaMask/metamask-extension/pull/4908/files#diff-04c6e90faac2675aa89e2176d2eec7d8R29).

Also note that latest npm version is 6.2.0 so the word "update" doesn't apply if you're running a newer version.

Reply:
I'll use the word 'Select' instead

Reply:
When I installed Node version 8.11.3 with `nvm` it did not update my `npm` version so I think it's necessary.